catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageWebConverter.java

```diff
@@ -0,0 +1,1760 @@
+                }
+            }
+
+            String name = getStringFromMap(NAME, map);
+            if (StringUtils.isNotBlank(name)) {
+                slot.setName(name);
+                populated = true;
+            }
+
+            if (populated) {
+                slots.add(slot);
+            }
+        }
+    }
+
+    private static void populateSlotStringValue(Map<String, Object> map, SlotType1 slot) {
+        ValueListType valueList = RIM_FACTORY.createValueListType();
+        valueList.getValue()
+                .addAll(getStringListFromMap(map, VALUE));
+
+        slot.setValueList(RIM_FACTORY.createValueList(valueList));
+    }
+
+    private static boolean populateSlotGMPointValue(Map<String, Object> map, SlotType1 slot) {
+        boolean populated = false;
+        PointType point = GML_FACTORY.createPointType();
+
+        if (map.containsKey(VALUE)) {
+            Map<String, Object> valueMap = (Map<String, Object>) map.get(VALUE);
+
+            if (valueMap.containsKey(POINT_KEY)) {
+                Map<String, Object> pointMap = (Map<String, Object>) valueMap.get(POINT_KEY);
+
+                BigInteger dimension = getBigIntFromMap(SRS_DIMENSION, pointMap);
+                if (dimension != null) {
+                    point.setSrsDimension(dimension);
+                    populated = true;
+                }
+
+                String valueToPopulate = getStringFromMap(SRS_NAME, pointMap);
+                if (StringUtils.isNotBlank(valueToPopulate)) {
+                    point.setSrsName(valueToPopulate);
+                    populated = true;
+                }
+
+                valueToPopulate = getStringFromMap(POSITION, pointMap);
+                if (StringUtils.isNotBlank(valueToPopulate)) {
+                    String[] values = StringUtils.split(valueToPopulate);
+
+                    DirectPositionType directPosition = GML_FACTORY.createDirectPositionType();
+
+                    for (String value : values) {
+                        directPosition.getValue()
+                                .add(Double.valueOf(value));
+                    }
+
+                    point.setPos(directPosition);
+                    populated = true;
+                }
+
+                if (populated) {
+                    AnyValueType anyValue = WRS_FACTORY.createAnyValueType();
+                    anyValue.getContent()
+                            .add(GML_FACTORY.createPoint(point));
+
+                    net.opengis.cat.wrs.v_1_0_2.ValueListType valueList =
+                            WRS_FACTORY.createValueListType();
+                    valueList.getAnyValue()
+                            .add(anyValue);
+
+                    slot.setValueList(RIM_FACTORY.createValueList(valueList));
+                }
+            }
+
+        }
+
+        return populated;
+    }
+
+    private static List<String> getStringListFromMap(Map<String, Object> map, String key) {
+        if (!map.containsKey(key)) {
```

Comment:
Should assign `map.get(key)` to a variable and use that variable to do the instance checks and add to the lists instead of calling `map.get(key)` 4 times.